app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,32 @@
+class OrdersController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_order, only: [:show]
+
+  def new
+    @order = Order.new
+  end
+
+  def create
+    @order = Order.new(user: current_user, total: current_user.cart.total_cart_price)
+    if @order.save
+      @order.purchase_line_items
+      redirect_to order_path(@order), notice: "Your order has been completed"
```

Comment:
The show method is set in the before_action :set_order, and is otherwise empty so I removed it but made sure to include it in my routes. It is the confirmation page after successfully completing an order.